sound/soc/sof/intel/intel_ext_manifest.h

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: ((GPL-2.0 WITH Linux-syscall-note) OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2020 Intel Corporation. All rights reserved.
+ */
+
+/*
+ * Intel extended manifest is a extra place to store intel specific
+ * metadata about firmware, for example LPRO/HPRO configuration is
+ * intel cavs specific.
+ * This part of output binary is not signed.
+ */
+
+#ifndef __INTEL_CAVS_EXT_MANIFEST_H__
+#define __INTEL_CAVS_EXT_MANIFEST_H__
+
+#include <sound/sof/ext_manifest.h>
+
+/* EXT_MAN_ELEM_PLATFORM_CONFIG_DATA elements identificators */
+enum sof_cavs_config_elem_type {
+	SOF_EXT_MAN_CAVS_CONFIG_CAVS_LPRO	= 1,
+	SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM,	/*< keep it at the end of enum list */
+};
+
+/* EXT_MAN_ELEM_PLATFORM_CONFIG_DATA elements */
+struct sof_ext_man_cavs_config_data {
+	struct sof_ext_man_elem_header hdr;
+
+	struct sof_config_elem elems[SOF_EXT_MAN_CAVS_CONFIG_LAST_ELEM];
+} __packed;
+
+#endif /* __INTEL_CAVS_EXT_MANIFEST_H__ */
```

Comment:
@fredoh9 This probably sould go to same place as ext_manifest.h (i.e. to include/sound/sof/)

Reply:
This is the direction after review. Intel platform specifc ext manifest to separate file.

Reply:
I think he means intel_ext_manifest.h in include/sound/sof

Reply:
Ah, sorry for misunderstanding. However this file only used in hda-loader.c and intel specific. I don't think include/sound/sof is better place.